README.rst

```diff
@@ -112,6 +114,210 @@ Usage example
 Output example
 --------------
 
+
+The output of rantanplan is a complex structure that will be broke down for clarity.
+
+First of all, rantanplan will show a list of stanzas. Each stanza is then showns as two separate lists. A list of tokens, and a list of "phonological groups" i.e., the phonological units that form a verse after synalephas and sinaereris are taken into acount.
```

Comment:
Use just `First,` better. And capitalize `rantanplan`. Please, check for other typos (e.g, `showns`, `acount`)

Reply:
Fixed all typos I found.